### ISSUE TITLE
TypeError when starting Vite

### ISSUE THREAD

**Issue by mikewink** [defstudio/vite-livewire-plugin#50]
Hello Team defstudio,

first, I would like to say thank you for this great plugin! 🏆

When I follow the installation steps for the plugin, I get this error message back:

```bash
failed to load config from /Users/BOB/Dev/PHP/Laravel/PROJECT/vite.config.js
error when starting dev server:
TypeError: livewire is not a function
    at file:///Users/BOB/Dev/PHP/Laravel/PROJECT/vite.config.js.timestamp-1683899540373-e3c526847558a.mjs:14:5
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
```

I have another (older) Laravel project that works just fine with the given instructions. I noticed that `package.json` now is of `type=module` and that the `postcss.config` is now `export default {` instead of `module.exports = {`, but I do not know if that is the reason. 

My current vite.config.js looks like this:

```js
import {defineConfig} from 'vite';
import laravel from 'laravel-vite-plugin';
import livewire from '@defstudio/vite-livewire-plugin';

export default defineConfig({
    plugins: [
        laravel({
            input: [
                'resources/css/app.css',
                'resources/js/app.js',
            ],
            refresh: false,
        }),
        livewire({
            refresh: ['resources/css/app.css'],
        }),
    ],
});
```

Package.json

```json
{
    "private": true,
    "type": "module",
    "scripts": {
        "dev": "vite",
        "build": "vite build"
    },
    "devDependencies": {
        "@alpinejs/focus": "^3.10.5",
        "@defstudio/vite-livewire-plugin": "^1.1.0",
        "@tailwindcss/forms": "^0.5.2",
        "@tailwindcss/typography": "^0.5.0",
        "alpinejs": "^3.0.6",
        "autoprefixer": "^10.4.7",
        "axios": "^1.1.2",
        "laravel-vite-plugin": "^0.7.5",
        "postcss": "^8.4.14",
        "tailwindcss": "^3.1.0",
        "vite": "^4.0.0"
    }
}
```

Laravel: 10.10.1
Jetstream: 3.2.0
Livewire: 2.12.3
Alpine.js: 3.12.1

**Comment by fabio-ivona** [comment]
Hi we'll try to reproduce this error, thanks for pointing it out

**Comment by bnzo** [comment]
This is because of the module type added in https://github.com/laravel/laravel/pull/6090

It should work again if you modify your `package.json`

```json
{
    "private": true,
    "type": "commonjs",
    "scripts": {
        "dev": "vite",
        "build": "vite build"
    },
    "devDependencies": {
        "@alpinejs/focus": "^3.10.5",
        "@defstudio/vite-livewire-plugin": "^1.1.0",
        "@tailwindcss/forms": "^0.5.2",
        "@tailwindcss/typography": "^0.5.0",
        "alpinejs": "^3.0.6",
        "autoprefixer": "^10.4.7",
        "axios": "^1.1.2",
        "laravel-vite-plugin": "^0.7.5",
        "postcss": "^8.4.14",
        "tailwindcss": "^3.1.0",
        "vite": "^4.0.0"
    }
}
```

and also your `postcss.config.js`

```js
module.exports = {
    plugins: {
        tailwindcss: {},
        autoprefixer: {},
    },
};
```

**Comment by ezehkingsleyuchenna** [comment]
@bnzo I tried to update the package.json like you said. I ran into another error 

_(node:11064) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
[Failed to load PostCSS config: Failed to load PostCSS config (searchPath: C:/laragon/www/...): [SyntaxError] Unexpected token 'export'
C:\laragon\www\...\postcss.config.js:1
export default {_

I tried changing the vite.config.js extension to .mjs and it is working.
